_drafts/bridging-digital-and-physical.markdown

```diff
@@ -7,63 +7,63 @@ image:        http://image.shutterstock.com/display_pic_with_logo/1451378/316560
 
-I discussed what to print with my ping-pong hitting partner, Alex. He builds our iOS apps. Should I print an Eiffel Tower? Should I print a phone case? Should I try modeling something myself, and if so, what? How many tiny TIE Interceptors could I print with three half-used large spools, and would that be enough to fend off tiny impending rebel assaults?
+I discussed what to print with my ping-pong hitting partner, Alex. He builds our iOS apps. Should I print an Eiffel Tower? Should I print a phone case? Should I try modeling something myself, and if so, what? How many tiny TIE Interceptors could I print with three half-used large spools, and would that be enough to fend off tiny rebel assaults?
 
-We decided to try printing a ping-pong paddle. We downloaded ScurtWad's [model](http://www.thingiverse.com/thing:175326), exported the print file from MakerWare at the default settings, and pressed print. Our eyes filled with wonder as the bot sprang to life. 
+We decided to try printing a ping-pong paddle. We exported ScurtWad's [model](http://www.thingiverse.com/thing:175326) from MakerWare at the default settings, and pressed print. Our eyes filled with wonder as the bot sprang to life.
 
 Our first print was an epic fail. We refer to it as the ramen paddle.
 
 ![ramen paddle](/images/makerbot/failed_paddle.png)
 
-The problem was that the model, oriented like this, needs supports. Without supports, the extruder extrudes into empty space. Who knew? It filled that empty space with a spaghetti-like structure. 
+The problem was that the model, oriented like this, needs supports. Without supports, the extruder extrudes into empty space. Who knew? It filled that empty space with a spaghetti-like structure.
 
 ### First Steps
 
-We had a couple laughs, played a round or two with what is arguably the worst paddle ever made, and then tried again. I checked the "supports" checkbox, re-exported the print file, and fired up the bot. 
+We had a couple laughs, played a round or two with the worst paddle ever made, and then tried again. I checked the "supports" checkbox, re-exported the print file, and fired up the bot.
 
 ![grey paddle printing](/images/makerbot/grey_paddle_printing.jpg)
 
 This one came out better. But the side we put supports on was too rough, and it's a little egg-shaped:
 
 ![grey paddle rough](/images/makerbot/grey_paddle_rough.jpg)
 
-Will suggested modifying the model to make it rounder, and orienting it diagonally on the build plate so it would need fewer supports and they'd attach to the side rather than the face. Instead, we decided to stop printing ping-pong paddles. 
+Will made two suggestions. First, alter the model to make it rounder. And second, and orient it on the build plate diagonally. By doing so, it would need fewer supports, and they would attach to the side rather than the face. Instead of following his advice, we decided to stop printing ping-pong paddles.
```

Comment:
"Will made two suggestions: first, alter the model to make it rounder; second, orient it on the build plate diagonally."